pkg/antctl/raw/multicluster/commands.go

```diff
@@ -25,8 +29,37 @@ var GetCmd = &cobra.Command{
 	Short: "Display one or many resources in a ClusterSet",
 }
 
+var CreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create multi-cluster resources",
+}
+
+var AddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add new member cluster to a ClusterSet",
+}
+
+var DeleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete resources in a ClusterSet",
+}
+
+var DeployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Deploy the leader cluster or member cluster",
```

Comment:
Deploy Antrea Multi-cluster to a leader or member cluster

Reply:
done